jme3-core/src/main/java/com/jme3/shadow/DirectionalLightShadowRenderer.java

```diff
@@ -151,16 +162,16 @@ protected void updateShadowCams(Camera viewCam) {
         ShadowUtil.updateFrustumPoints(viewCam, frustumNear, zFar, 1.0f, points);
 
         shadowCam.setFrustumFar(zFar);
-        shadowCam.getRotation().lookAt(light.getDirection(), shadowCam.getUp());
+        shadowCam.getRotation().lookAt(light.getDirection(), shadowCam.getUp(tempVec));
```

Comment:
This is only called once per frame, is it really worth it to create a reusable vector? (Did this come from profiling and finding lots of memory allocations?)